.github/workflows/docker-push-to-registries.yml

```diff
@@ -76,14 +76,27 @@ on:
         default: temurin
         description: |
           Specifies the JDK to use, defaults to temurin.
+      ALLOW_MUTABLE_TAGS:
+        required: false
+        type: boolean
+        default: true
+        description: |
+          Set `false` to avoid adding mutable tags like `latest` and `development` e.g. if AWS ECR has mutable tags disabled, and 400s
+      package-directory:
```

Comment:
At the risk as of being incredibly nit picking, all the other inputs are upper case, but this one is lower case

Reply:
[TELFE-691](https://telicent.atlassian.net/browse/TELFE-691)